modules/home/ui/navbar.tsx

```
@@ -12,6 +15,7 @@ import {
 import Link from "next/link";
 
 export const Navbar = () => {
+  const isMobile = useIsMobile()
```

ESLint

[error] 'isMobile' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)